src/display/annotation_layer.js

```diff
@@ -718,6 +724,78 @@ var StrikeOutAnnotationElement = (
 })();
 
 /**
+ * @class
+ * @alias FileAttachmentAnnotationElement
+ */
+var FileAttachmentAnnotationElement = (
+    function FileAttachmentAnnotationElementClosure() {
+  function FileAttachmentAnnotationElement(parameters) {
+    var isRenderable = !!(parameters.data.hasPopup ||
+                          parameters.data.title || parameters.data.contents);
```

Comment:
I just thought of something here: Is it actually necessary for a `FileAttachment` to have either a Popup (or title/contents), or should it be rendered regardless?
For other types of annotations this kind of `isRenderable` check makes sense, but for `FileAttachment` it seems that it should be possible to download the attached file regardless!

If we change this, you probably want to change line https://github.com/mozilla/pdf.js/pull/6988/files#diff-1097e3b131ba22f8e8a6c2f20e3ab5ecR758 to:
```javascript
if (!this.data.hasPopup && (this.data.title || this.data.contents)) {
```